ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift

```diff
@@ -42,13 +53,28 @@ final class SidebarViewController: UITableViewController {
 
         switch switchType {
         case .timeline:
-            rootViewController?.navigationDrawerController?.toggleLeftView()
+            if let _ = UIApplication.topViewController() as? FilterViewController {
+                navigationDrawerController?.toggleLeftView()
+            } else {
+                let vc = FilterViewController()
+                rootViewController?.pushViewController(vc, animated: true)
+                rootViewController?.navigationDrawerController?.toggleLeftView()
+            }
+
+
         case .about:
             break
         case .info:
             break
         case .map:
-            break
+            if let _ = UIApplication.topViewController() as? FloorMapViewController {
```

Comment:
After fixing [this issue](https://github.com/DroidKaigi/conference-app-2020/issues/674), we don't need to judge showing `ViewController` with this way. 
Would you fix this code like [this](https://github.com/DroidKaigi/conference-app-2020/pull/672/files#diff-34264a1af610b65a3b561894e675d97eR47)?
```swift
rootViewController.viewControllers.first is AboutViewController
```

And please use `setViewControllers` to move `ViewController` 🙏

Reply:
I'm sorry.. I reviewed about this https://github.com/DroidKaigi/conference-app-2020/pull/624#discussion_r371607053 🙇

Reply:
Sorry, you don't have to do about this 🙇